pom.xml

```diff
@@ -4,7 +4,7 @@
   <parent>
     <groupId>org.nuxeo</groupId>
     <artifactId>nuxeo-parent</artifactId>
-    <version>2023.0</version>
+    <version>2023.16</version>
```

Comment:
While we're at upgrading, I think you could align here on the latest 2023 version, `2023.30`